### PR TITLE
Update xdebug version to support PHP 8.1

### PR DESCRIPTION
**Pull request by martin-cod**
- [ ] There is an issue ticket which accompanies my PR.
- [x] I have followed the [guidelines for contributing](https://github.com/weprovide/valet-plus/blob/master/CONTRIBUTING.md).
- [x] I have checked that there aren't other open [pull requests](https://github.com/weprovide/valet-plus/pulls) for the same issue ticket.
- [x] I have formatted my code with the [PSR-2](http://www.php-fig.org/psr/psr-2/) coding style before submitting my PR.
-----

**I have read the contribution guidelines and am targeting the branch `release/2.4`:**  
Because this is a Bug Fix to support changes provided in `release/2.4` branch

### CODE REVIEW

File: cli/Valet/Pecl.php [hunk 1]
```diff
@@ -43,10 +43,10 @@ class Pecl extends AbstractPecl
...
     const EXTENSIONS = [
         self::XDEBUG_EXTENSION => [
-            '8.1' => '3.0.4',
-            '8.0' => '3.0.4',
-            '7.4' => '3.0.4',
-            '7.3' => '3.0.4',
+            '8.1' => '3.1.5',
+            '8.0' => '3.1.5',
+            '7.4' => '3.1.5',
+            '7.3' => '3.1.5',
```
Comment (rossmc) [inline cli/Valet/Pecl.php:49]: Thanks @martin-cod 

For PHP 7.3 & 7.4 we should use xdebug v2.9.8 as Magento 2.3 does not support these versions.

REF: https://github.com/weprovide/valet-plus/issues/580#issuecomment-886844867

cc @mischabraam

Comment (rossmc) [reply cli/Valet/Pecl.php:49]: I tested this locally, by using `3.1.5` on PHP 8 and `2.9.8` on PHP 7, it works perfectly

CleanShot 2022-07-26 at 15 27 51@2x

**Comment by mischabraam** [comment]
Updating xdebug is fine, but 3.x is currently not supported, see https://github.com/weprovide/valet-plus/blob/release/2.4/cli/stubs/z-performance.ini
xdebug 3 needs other config then 2. And we can't have both configs.

File: cli/Valet/Pecl.php (hunk 1 above)
Comment (rossmc) [reply cli/Valet/Pecl.php:49]: For anyone needing it, with the above changes to [cli/Valet/Pecl.php](https://github.com/martinboy/valet-plus/blob/67509c146ffcc5cf9d6b54b9ace5f1dab45b9d3a/cli/Valet/Pecl.php#L45-L50) you can set your xdebug version as needed.
i.e. `~/.composer/vendor/weprovide/valet-plus/cli/Valet/Pecl.php`

For other config changes which are needed for refernce see [z-performance.ini#L64-L78](https://github.com/weprovide/valet-plus/blob/691ae2fce7b463b369d522db59412186efa390dd/cli/stubs/z-performance.ini#L64-L78)

The path to this file on your local system can be found in [phpinfo](https://www.php.net/manual/en/function.phpinfo.php).

This way you will be able to use xdebup on both Magento 2.3 and 2.4.

Comment (martin-cod) [reply cli/Valet/Pecl.php:49]: @rossmc thanks for checking this. On my end I tested xdebug 3.1.5 on Magento 2.4.4 and PHP 7.4. It worked fine. Didn't check Magento 2.3

**Pull request merged**